langs/python.go

```diff
@@ -32,7 +34,14 @@ func (h *PythonLangHelper) GenerateBoilerplate(path string) error {
 		return err
 	}
 	depFile := "requirements.txt"
-	if err := ioutil.WriteFile(depFile, []byte(reqsPythonSrcBoilerplate), os.FileMode(0644)); err != nil {
+	fdkVersionInfo, err := h.GetLatestFDKVersion()
+	if err != nil {
+		fmt.Println("Unable to get latest FDK version, using default")
```

Comment:
Where are we setting fdkVersionInfo to default if err was not nil ?

Reply:
@metamemelord

Reply:
We go via the current flow: assign `fdk` in requirements.txt and not add version. On the other hand, if we do get the latest version, we add `fdk>=<version>` in requirements.txt.

Reply:
Failure to fetch latest FDK version can mean one of two things:

1. Pypi is down, in this case user function will not deploy - this is sort unrecoverable at the moment.
2. Pypi changes response json. In this case, we have 2 choices: prevent user to deploy function altogether until we fix the parsing logic OR leave the fdk tag empty in requirements.txt and let pip pick latest automatically. Current flow takes the latter path.